EIPS/eip-2981.md

```diff
@@ -12,33 +12,35 @@ requires: 165
 
 ## Simple Summary
 
-A standardized way to retrieve royalty payment information for ERC-721 tokens to enable universal support for royalty payments in all NFT marketplaces and ecosystem participants.
+A standardized way to retrieve royalty payment information for NFT tokens to enable universal support for royalty payments in all NFT marketplaces and ecosystem participants.
 
 ## Abstract
 
-This standard extends the [ERC-721 specification](./eip-721.md) to enable setting a royalty amount paid to the NFT creator or rights holder every time an NFT is sold and re-sold. This is intended for NFT marketplaces that want to support the ongoing funding of artists and other NFT creators. The royalty payment must be voluntary as required by the EIP-721 standard, as `transferFrom()` includes NFT transfers between wallets, and executing `transferFrom()` does not always imply a sale occurred. Marketplaces and individuals implement this standard by retrieving the royalty payment information with `royaltyInfo()`, which specifies how much to pay to which address for a given sale price. The exact mechanism for paying and notifying the recipient will be defined in future EIPs. This ERC should be considered a minimal, gas-efficient building block for further innovation in NFT royalty payments.
+This standard extends NFT standards such as the [ERC-721](./eip-721.md) and [ERC-1155]](./eip-1155.md) specifications to enable setting a royalty amount paid to the NFT creator or rights holder every time an NFT is sold and re-sold. This is intended for NFT marketplaces that want to support the ongoing funding of artists and other NFT creators. The royalty payment must be voluntary, as `transferFrom()` includes NFT transfers between wallets, and executing `transferFrom()` does not always imply a sale occurred. Marketplaces and individuals implement this standard by retrieving the royalty payment information with `royaltyInfo()`, which specifies how much to pay to which address for a given sale price. The exact mechanism for paying and notifying the recipient will be defined in future EIPs. This ERC should be considered a minimal, gas-efficient building block for further innovation in NFT royalty payments.
```

Comment:
>This standard _extends_ NFT standards such as the [ERC-721](./eip-721.md) and [ERC-1155]](./eip-1155.md) specifications to _enable setting a royalty amount_ paid to the NFT creator or rights holder every time an NFT is sold and re-sold.'

I suggest:

''This standard allows contracts, such as NFTs that support [ERC-721](./eip-721.md) and [ERC-1155]](./eip-1155.md) interfaces, to signal a royalty amount to be paid to the NFT creator or rights holder every time the NFT is sold or re-sold.''

Rationale:

'extends' gives the impression that there is some dependency on NFT standards. That constraint has been removed form the backward compatibility section. The only crossover is the notion of indexing within the contract.

'setting' is not really the the function of this standard, it is about 'signaling'

Reply:
I agree about the signalling part as well 👍🏻